projects/plugins/jetpack/extensions/blocks/subscriber-login/edit.js

```diff
@@ -38,17 +43,26 @@ function SubscriberLoginEdit( { attributes, setAttributes, className } ) {
 							id={ logOutInputId }
 						/>
 					</BaseControl>
+					<ToggleControl
+						label={ __( 'Manage subscription link', 'jetpack' ) }
```

Comment:
"Show Manage subscription link" instead? A bit more descriptive.

Reply:
True! Thinking if any difference between:

- Show Manage subscription link
- Show manage subscription link
- Show "manage subscription" link

<img width="294" alt="image" src="https://github.com/Automattic/jetpack/assets/87168/800a7bb1-1d70-4d63-b0c9-49f13123b9dd">

@crisbusquets thoughts?

Reply:
I'm leaning towards this: `Show "manage subscription" link` because it seems easier to read. But does it fit in one line in English?

Other languages will probably need two lines. Spanish would be something like `Mostrar el enlace "administrar suscripción"`.

Reply:
Or what about: `Show link to manage subscription`? Although it's a bit longer 😅

Reply:
I would go for `Show manage subscription link` or `Show "Manage subscription" link`. The first option looks simpler and it's shorter as well.

Reply:
Let's go with `Show "Manage subscription" link` 👍